src/styles/tailwind-base.css

```diff
@@ -33,3 +33,4 @@
 }
 
 @tailwind utilities;
+@tailwind components;
```

Comment:
Why this? @roushou 

We have not see the need to have those in a Tailwind application.

Reply:
Right it's not needed here. I added it because I'm used to have it and I often define custom classes in `@layer components` but there's none in OnchainKit.